Sources/BraintreePayPal/BTPayPalRequest.swift

```diff
@@ -115,10 +99,8 @@ import BraintreeCore
         landingPageType: BTPayPalRequestLandingPageType = .none,
         displayName: String? = nil,
         merchantAccountID: String? = nil,
-        lineItems: [BTPayPalLineItem]? = nil,
```

Comment:
I think this is available for both checkout and vault. We can keep line items here and pass it through on the super inits.